### PR TITLE
fix bugs that propagated to the registry

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -11,8 +11,7 @@ Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 
 [compat]
-Julia = "=1.0"
-SymPy = ">=0.8"
+julia = "=1.0"
```
Comment (fredrikekre) [inline Project.toml]: Are you sure this is what you want? This package does not work on Juila 1.1?

Comment (fredrikekre) [reply Project.toml]: It seems to pass tests, so I am assuming what you really want is
```
julia = "1"
```

Comment (fredrikekre) [inline Project.toml]:
```suggestion
julia = "1"
```

**Comment by timkittel** [comment]
Yes, @fredrikekre. Thanks a lot. And your assumption was spot on.

**Pull request merged**